broker/executor.go

```diff
@@ -56,7 +56,7 @@ func (qe *queryExecutorImpl) Execute(ctx context.Context, sqlQuery string, w htt
 	qc := &query.AQLQueryContext{
 		Query: aqlQuery,
 	}
-	qc.Compile(qe.tableSchemaReader)
+	qc.Compile(qe.tableSchemaReader, topology.NewTopologyShardOwner(qe.topo))
```

Comment:
heads up: we won't need this after my PR merged

Reply:
ok. can remove that during the merge